tests/zfs-tests/tests/functional/mmp/mmp_on_uberblocks.ksh

```diff
@@ -37,43 +40,43 @@
 verify_runnable "both"
 
 UBER_CHANGES=0
+EXPECTED=$(($(echo $DISK | wc -w) * 10))
+FUDGE=$((EXPECTED * 20 / 100))
+MIN=$((EXPECTED - FUDGE))
+MAX=$((EXPECTED + FUDGE))
+KSTAT_PATH="/proc/spl/kstat/zfs/$TESTPOOL/multihost"
```

Comment:
nit: mmp_reset_interval.ksh and mmp_write_uberblocks.ksh also were updated to depended on the history. Why don't we add this path as a variable in mmp.cfg called `MMP_HISTORY_PATH`.